storage-api/src/java/org/apache/hadoop/hive/ql/io/sarg/SearchArgument.java

```diff
@@ -162,6 +162,11 @@ public boolean isNeeded() {
    */
   public ExpressionTree getExpression();
 
+  /**
+   * Get the expression tree without the
+   */
+  public ExpressionTree getUnexpandedExpression();
```

Comment:
I find `unexpanded` a little strange - a different keyword might fit this thing better (what do you think about compact or  original).